docs/Aurora/Spark job submit tool usage on aurora.md

```diff
@@ -0,0 +1,271 @@
+----------------  ---------------- ----------------  -------- - -----
+27643.amn-0001    start-spark.sh   damon                    0 E lustre_scaling
+```
+To check your job status periodically, you can use watch command. After the job ends, you can use Ctrl-C to stop "watch" process.
+```shell
+$ watch -n 2 qstat 27643.amn-0001
+Job id            Name             User              Time Use S Queue
+----------------  ---------------- ----------------  -------- - -----
+27643.amn-0001    start-spark.sh   damon                    0 Q lustre_scaling
+```
+
+### Check Output of Your Job
+The output of your job is stored in your working directory and here is an example.
+```text
+spark_work_home
+├── 27643.amn-0001/
+│     └── conf/                         // Spark conf like spark-env.sh, spark-default.xml and slaves
+│     └── logs/                         // Spark master and worker logs
+│     └── app-20230301080732-0000/      // After Spark submitted successfully, you will get application ID that contains stderr and stdout log for each executor.
+├── 27643.amn-0001.ER                   // All stderr output for your job.
+└── 27643.amn-0001.OU                   // All stdout output for your job.
+```
+
+The results of dense K-Means example are at the end of **27643.amn-0001.OU**.
+
+### Command Line Help
+submit-spark.sh provides a few arguments. You can get detailed info about the arguments and examples with below command.
+```shell
+$ /lus/flare/projects/Aurora_deployment/spark/spark-job/bin/submit-spark.sh -h
+Spark Job Submit Tool v1.0.3 for Sunspot
+ 
+Usage:
+    submit-spark.sh [options] JOBFILE [arguments ...]
+ 
+JOBFILE can be:
+    script.py                       pyspark scripts
+    run-example examplename         run Spark examples, like "SparkPi"
+    --class classname example.jar   run Spark job "classname" wrapped in example.jar
+    shell-script.sh                 when option "-s" is specified
+ 
+Required options: 
+    -l walltime=<v>                 Max run time, e.g., -l walltime=30:00 for running 30 minutes at most
+    -l select=<v>                   Job node count, e.g., -l select=2 for requesting 2 nodes
+    -q QUEUE                        Queue name
+ 
+Optional options:
+    -A PROJECT                      Allocation name
+    -o OUTPUTDIR                    Directory for job output files (default: current dir)
+    -s                              Enable shell script mode
+    -y                              Spark master uses a separate node
+    -I                              Start an interactive ssh session
+    -b                              Prefer Spark built-in mllib to default OAP mllib
+    -n                              Run without DAOS loaded
+    -h                              Print this help messages
+Example:
+    submit-spark.sh -A Aurora_deployment -l walltime=60 -l select=2 -q workq kmeans-pyspark.py daos://pool0/cont1/kmeans/input/libsvm 10
```

Comment:
I saw `daos=default` in earlier example  but didn't see this option explained in this helper message or the example here. can you please explain and add? 
Let's also make sure all examples below work.